src/appleseed.shaders/src/appleseed/as_globals.osl

```diff
@@ -45,104 +45,119 @@ shader as_globals
     point Pref = P
     [[
         int lockgeom = 0,
-        string widget = "null"
+        string widget = "null",
+        int as_max_param_id = 0
     ]],
     normal Nref = N
     [[
         int lockgeom = 0,
-        string widget = "null"
+        string widget = "null",
+        int as_max_param_id = 1
     ]],
     output point out_world_P = P
     [[
         string as_maya_attribute_name = "worldP",
         string as_maya_attribute_short_name = "wp",
-        string label = "Surface Position"
+        string label = "Surface Position",
+        int as_max_param_id = 2
```

Comment:
So all `as_max_param_id` on output parameters have the same value?

Reply:
Yes, and the reason is because all the outputs go under single multi-string combo-box parameter in max paramblock.
It is probably enough to set it once in one output but I didn't check that option.

Reply:
That makes sense, thanks for confirming.